onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -332,11 +332,11 @@ including arg name, arg type (contains both type and shape).)pbdoc")
           "node shape (assuming the node holds a tensor)");
 
   py::class_<SessionObjectInitializer>(m, "SessionObjectInitializer");
-  py::class_<InferenceSession>(m, "InferenceSession", R"pbdoc(This is the main class used to run a model.)pbdoc")
+  py::class_<Session>(m, "Session", R"pbdoc(This is the main class used to run a model.)pbdoc")
```

Comment:
>Session [](start = 13, length = 7)

python binding still has some issue, I'll fix it in next iteration.

Reply:
Fixed the python binding errors.
@xadupre could you please help review as well?

Reply:
Change my approach and py binding don't have to change.